microsoft-edge/extensions/publish/microsoft-edge-badge.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Use the Get it from Microsoft Edge badge to promote your extension
+description: Use the Get it from Microsoft Edge badge in your websites, documentation, or tutorials when publishing an extension, theme, or web app at the Microsoft Edge Add-ons site.  # key words before col 158
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/13/2025
+---
+<!-- todo: in desc above, web app ok? -->
+# Use the Get it from Microsoft Edge badge to promote your extension
+<!-- todo: note title of upstream article, toc bucket/nav, & docset name  https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines -->
+
+<!-- todo: in this article, globally delete "add-ons" -->
+
+To promote your Microsoft Edge extension, theme, or web app, use the Get it from Microsoft Edge badge in your websites, documentation, tutorials, and other services or integrations.  This ensures consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.
+
+The Get it from Microsoft Edge badge is a visual asset designed to help you promote your Microsoft Edge add-ons and build user trust.
+
+![The "Get it from Microsoft Edge" badge](./microsoft-edge-badge-images/get-it-from-microsoft-edge-badge.png)
+
+The Get it from Microsoft Edge badge appears in full color with a dark background, and includes a gray border and transparent corners.  The badge `.png` files are provided in 41 languages.
+
+The Get it from Microsoft Edge badge lets you showcase your Microsoft Edge add-ons with Microsoft Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Get it from Microsoft Edge badge is intended exclusively for promoting add-ons that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.
+
+The generic Microsoft Store badge reads **Get it from Microsoft**:
+
+![The generic Microsoft Store badge](./microsoft-edge-badge-images/generic-microsoft-store-badge.png)
+
+Unlike generic Microsoft [Store badges](/windows/apps/publish/app-marketing-guidelines#store-badges),<!-- todo: png showing "generic" Microsoft Store badge --> the Get it from Microsoft Edge badge provides a distinct, trusted visual cue that connects your Edge add-ons directly to the Microsoft Edge ecosystem.
+
+The Get it from Microsoft Edge badge is not the same as the Microsoft brand logo, and they cannot be used interchangeably.
+
+
+<!-- ====================================================================== -->
+## Download the badge image files
+
+Download the complete set of badge `.png` files.
+
+<!-- todo: upload the .zip file to either:
```

Comment:
1. Delete the macOS hidden folder from the .zip file that contains the badge .png files.
2. Upload the .zip file to download.microsoft.com (www.microsoft.com/download).

@captainbrosset posted in a forum on Monday, Aug. 25 asking how to do this.
@G-u-r-j-e-e-t